bamboo/common_python/tools.py

```diff
@@ -60,7 +62,7 @@ def get_command(cluster,
         data_filename_test_default, data_reader_name, data_reader_path,
         data_reader_percent, exit_after_setup, metadata, mini_batch_size,
         model_folder, model_name, model_path, num_epochs, optimizer_name,
-        optimizer_path, processes_per_model,
+        optimizer_path, processes_per_model, restart_dir,
```

Comment:
Specifically does `disable_cuda` need to appear here.

Reply:
@forsyth2 I have no idea. Things are working with it the way it is, but perhaps I've done something wrong (and true to Python, it works anyway...).